src/Layers/xrRenderPC_GL/glTexture.cpp

```diff
@@ -209,8 +163,7 @@ GLuint CRender::texture_load(LPCSTR fRName, u32& ret_msize, GLenum& ret_desc)
 
         xr_strlwr(fn);
         ret_desc = target;
-        img_loaded_lod = is_target_cube(texture.target()) ? img_loaded_lod : get_texture_load_lod(fn);
```

Comment:
Why you removed it?

Reply:
`get_texture_load_lod` called `strstr` only to determine the LOD, in order to eventually find out the size of the texture loaded into memory! At the same time, GLI can determine the size of the loaded texture taking into account the LOD. Therefore, it’s impossible to transfer exactly this edit to DirectX - there you have to calculate in the old way.

Reply:
You're talking about `img_loaded_lod`, but I'm talking about `strstr`.

Reply:
Just open dx10SH_Texture.cpp